### PR TITLE
Update README-zh.md

### PR DESCRIPTION
**Pull request by LJTian**
Added run 'make container-image' command timeout solution

### CODE REVIEW

File: README-zh.md [hunk 1]
```diff
@@ -257,6 +257,49 @@ This works for Catalina as well as Mojave macOS.
 -->
 这适用于 Catalina 和 Mojave macOS。
 
+### 对执行 make container-image 命令部分地区访问超时的故障排除
+
+现象如下:
```
Comment (Sea-n) [inline README-zh.md:262]:
```suggestion
现象如下：
```

File: README-zh.md [hunk 2]
```diff
@@ -257,6 +257,49 @@ This works for Catalina as well as Mojave macOS.
...
+hugolib/integrationtest_builder.go:29:2: golang.org/x/tools@v0.1.11: Get "https://proxy.golang.org/golang.org/x/tools/@v/v0.1.11.zip": dial tcp 142.251.42.241:443: i/o timeout
+deploy/google.go:24:2: google.golang.org/api@v0.76.0: Get "https://proxy.golang.org/google.golang.org/api/@v/v0.76.0.zip": dial tcp 142.251.43.17:443: i/o timeout
+parser/metadecoders/decoder.go:32:2: gopkg.in/yaml.v2@v2.4.0: Get "https://proxy.golang.org/gopkg.in/yaml.v2/@v/v2.4.0.zip": dial tcp 142.251.42.241:443: i/o timeout
+The command '/bin/sh -c mkdir $HOME/src &&     cd $HOME/src &&     curl -L https://github.com/gohugoio/hugo/archive/refs/tags/v${HUGO_VERSION}.tar.gz | tar -xz &&     cd "hugo-${HUGO_VERS    ION}" &&     go install --tags extended' returned a non-zero code: 1
+make: *** [Makefile:69：container-image] error 1
+```
+
+请修改 `Dockerfile` 文件，为其添加网络代理。修改内容如下：
+
+```shell
```
Comment (Sea-n) [inline README-zh.md:279]:
````suggestion
```dockerfile
````

File: README-zh.md [hunk 3]
```diff
@@ -257,6 +257,49 @@ This works for Catalina as well as Mojave macOS.
...
+...
+FROM golang:1.18-alpine
+
+LABEL maintainer="Luc Perkins <lperkins@linuxfoundation.org>"
+
+ENV GO111MODULE=on                      # 需要添加内容1
+
+ENV GOPROXY=https://goproxy.cn,direct   # 需要添加内容2
```
Comment (Sea-n) [inline README-zh.md:287]: I think we shouldn't put unofficial in here.

File: README-zh.md [hunk 4]
```diff
@@ -257,6 +257,49 @@ This works for Catalina as well as Mojave macOS.
...
+    g++ \
+    musl-dev \
+    build-base \
+    libc6-compat
+
+ARG HUGO_VERSION
+...
+```
+
+**注意:** 此部分仅适用于中国大陆
```
Comment (Sea-n) [inline README-zh.md:301]:
```suggestion
**注意：** 此部分仅适用于中国大陆 
```

File: README-zh.md (hunk 3 above)
Comment (LJTian) [reply README-zh.md:287]: ENV GOPROXY=https://proxy.golang.org

Comment (LJTian) [reply README-zh.md:287]: /assign @Sea-n Looking forward to your reply

**Pull request merged**